### DISCUSSION TITLE
Aptos Testnet Archive public snapshot

### DISCUSSION THREAD

**Discussion by Igorarg91** [aptos-labs/aptos-developer-discussions#405] [Questions] [answered]
### Describe your question in detail.

i am looking for Aptos Testnet Archive public snapshot. I found pruned version, but would like to launch the archive node.

### Which operating system are you using?

N/A

### Which SDK or tool are you using? (if any)

N/A

**Comment by ying-w** [comment, marked as answer]
Google Bigquery has testnet data, you can find it here: `bigquery-public-data.crypto_aptos_testnet_us`

More info:
https://aptos.dev/en/build/apis/data-providers#google-bigquery-public-dataset
